File: tests/php/testcases/AddIntegersTest.php
<?
require_once 'base.php';

class AddIntegersTestCase extends TestServiceTestCase {

    public function test_null() {
        $this->assertEquals($this->service->add_integers(null, null), 0);
    }

    public function test() {
        $this->assertEquals($this->service->add_integers(100, 50), 150);
    }

}

File: tests/mono/testcases/AddIntegersTest.cs
using System;
using NUnit.Framework;

namespace TestCases {

    [TestFixture()]
    public class AddIntegersTest : TestServiceTest {

        [Test()]
        public void test_null() {
            Assert.AreEqual(0, service.add_integers(null, null));
        }

        [Test()]
        public void test_empty() {
            Assert.AreEqual(0, service.add_integers(0, 0));
        }

        [Test()]
        public void test() {
            Assert.AreEqual(150, service.add_integers(100, 50));
        }
    }
}
